charts/chainlink-cluster/templates/chainlink-deployment.yaml

```diff
@@ -0,0 +1,182 @@
+{{- range $cfg := .Values.chainlink.nodes }}
```

Comment:
We have a lot of duplication of charts between this and the qa-charts repo. Would it make sense to update the qa-charts to work with this as well? Otherwise when we need updates in one place now we need to make the same change in both.

Reply:
The idea is that we work on that charts to make them production-like and then reuse them from `core`. No need to have an additional repo for the official OSS chart that people can use to run some clusters.